src/ActionDropdown/ActionDropdown.stories.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import { ActionDropdown } from './ActionDropdown'
+
+const defaultAction = {
+  label: 'Medium',
```

Comment:
This feels like a possible area for confusion, two strings that are uppercase and sentence case. Maybe turn the value into a `const`?